content/ja/handbook/customer-success/csm/pto.md

```
@@ -0,0 +1,123 @@
+
+---
+
+GitLab の[有給休暇ポリシー](/handbook/people-group/time-off-and-absence/time-off-types/)の一環として、チームメンバーには積極的に休暇を取ることが奨励されています。しかし、顧客対応を行うチームメンバーにとっては、休暇を取ることが難しく感じることもあるため、このページでは CSM が定期的に休暇を取得し、燃え尽き症候群を避けながら顧客の成功を維持できるよう案内することを目的としています。
+
+## 休暇前に
+
+休暇を取得する際は、[PTO ポリシー](/handbook/people-group/time-off-and-absence/time-off-types/)のガイドラインに従い、Time Off by Deel に休暇を登録してください。
+
+休暇が始まる前に、以下の追加ガイドラインに従ってください。
+
+1. 不在中の[カバレッジを確立する](#establish-coverage)。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)